### DISCUSSION TITLE
Recommended way to clear cache

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6485] [Q&A] [answered]
Hi, can someone tell me the recommended way to clear all cache/cookies? I didn't find a command for that so I thought in deleting all files in `~/.local/share/qutebrowser/` and 
`~/.cache/qutebrowser`. Is this (manual deletion of files) the recommended way of doing so or is there a command for that?

**Comment by The-Compiler** [comment, marked as answer]
Yep, at the moment, the only way to do this is to delete those files by hand or deleting data for a specific website in `:devtools`. Also see #58. Note that `~/.local/share/qutebrowser/` also contains some qutebrowser state and e.g. your history, so you might want to be a bit more selective (or perhaps just delete the `webengine` folder in there).

**Reply by Ochiruko** [reply]
Is there any update on the cookies situation?

**Reply by The-Compiler** [reply]
No. There is an issue linked above you can subscribe to.
